sql/core/src/test/resources/sql-tests/results/show-create-table.sql.out

```diff
@@ -257,7 +257,7 @@ SHOW CREATE TABLE tbl
 -- !query schema
 struct<createtab_stmt:string>
 -- !query output
-CREATE TABLE `default`.`tbl` (
+CREATE TABLE default.tbl (
   `a` FLOAT,
```

Comment:
to be consistent, shall we also only quote the column name if needed? This is can be done in a new PR, as it's not related to v2 command migration.

Reply:
ok. I will do it.